### ISSUE TITLE
Invalid plugin descriptor for spotless-maven-plugin:1.25.0

### ISSUE THREAD

**Issue by pzygielo** [diffplug/spotless#470]
Previously - similar raised #388

- [x] summary of problem
```
[ERROR] Invalid plugin descriptor for com.diffplug.spotless:spotless-maven-plugin:1.25.0 (.../.m2/repository/com/diffplug/spotless/spotless-maven-plugin/1.25.0/spotless-maven-plugin-1.25.0.jar), Plugin's descriptor contains the wrong version: 1.24.3
```
- [x] gradle or maven version: maven
- [x] spotless version: 1.25.0 
- [x] operating system and version: N/A
- [x] copy-paste your full Spotless configuration block(s), and a link to a public git repo that reproduces the problem if possible: N/A
- [x] copy-paste the full content of any console errors emitted by `gradlew spotless[Apply/Check] --stacktrace`: N/A

**Comment by pzygielo** [comment]
Hmmm...
```
$ wget http://repo.maven.apache.org/maven2/com/diffplug/spotless/spotless-maven-plugin/1.24.3/spotless-maven-plugin-1.24.3.jar
$ wget http://repo.maven.apache.org/maven2/com/diffplug/spotless/spotless-maven-plugin/1.25.0/spotless-maven-plugin-1.25.0.jar
$ sha1sum spotless-maven-plugin-1.2*
f2faed4a554737fb771ef768e1c607be9db4432f  spotless-maven-plugin-1.24.3.jar
f2faed4a554737fb771ef768e1c607be9db4432f  spotless-maven-plugin-1.25.0.jar
```

**Comment by nedtwigg** [comment]
Fix published in `1.25.1`.

**Comment by pzygielo** [comment]
> Fix published in `1.25.1`.

Thanks!
